File: beginner/pagina02/1046/1046.php
<?php 

    list($start, $end) = explode(' ', readline());
    $start = (int) $start;
    $end = (int) $end;
    $time;

    if($start < $end){
        $time = $end - $start;
    } else {
        $time = (24 - $start) + $end;
    }

    echo "O JOGO DUROU {$time} HORA9(S)\n";

?>

File: beginner/pagina02/1046/1046.rs
use std::io;

fn main(){

    let mut input = String::new();
    io::stdin().read_line(&mut input).unwrap();

    let data: Vec<u32> = input.trim().split_whitespace().map(|x| x.parse().unwrap()).collect();
    let (a,  b) = (data[0], data[1]);
    let mut resp = 0;
    
    if a > b || a== b {
        resp = (24 - a) + b;
    } else {
        resp = b - a;
    }

    println!("O JOGO DUROU {} HORA(S)", resp);
}